blocks/api/raw-handling/utils.js

```diff
@@ -193,23 +110,16 @@ export function isEmpty( element ) {
 	} );
 }
 
+/**
+ * Checks wether HTML can be considered plain text. That is, it does not contain
+ * any elements that are not line breaks.
+ *
+ * @param {string} HTML The HTML to check.
+ *
+ * @return {boolean} Wether the HTML can be considered plain text.
+ */
 export function isPlain( HTML ) {
-	const doc = document.implementation.createHTMLDocument( '' );
-
-	doc.body.innerHTML = HTML;
-
-	const brs = doc.querySelectorAll( 'br' );
-
-	// Remove all BR nodes.
-	Array.from( brs ).forEach( ( node ) => {
-		node.parentNode.replaceChild( document.createTextNode( '\n' ), node );
-	} );
-
-	// Merge all text nodes.
-	doc.body.normalize();
-
-	// If it's plain text, there should only be one node left.
-	return doc.body.childNodes.length === 1 && doc.body.firstChild.nodeType === TEXT_NODE;
+	return ! /<(?!br)/i.test( HTML );
```

Comment:
`isPlain( '<brazenly-defying-odds-with-my-custom-element />' );`

Reply:
Oh no. :) I'll adjust and add a test case.

Reply:
😃